### DISCUSSION TITLE
When will blitz v1.0 be published?

### DISCUSSION THREAD

**Discussion by zwl1619** [blitz-js/blitz#2648] [Q&A]
When will blitz v1.0 be published?

**Comment by flybayer** [comment]
Can't promise an exact date, but I think likely a couple more months. Definitely making progress on it, just slower than expected :)

**Reply by reubence** [reply]
Hey, could you give us an update on this? Can we expect v1.0 to be deployed by EOY-2021?

**Reply by beerose** [reply]
Hey @reubence, we plan to have at least a release candidate around the end of 2021. You can follow the milestone: https://github.com/blitz-js/blitz/milestone/1 to see the progress

**Reply by zwl1619** [reply]
Hey @beerose could add https://github.com/blitz-js/blitz/issues/2506 to the milestone?

**Reply by beerose** [reply]
That's something that we'll focus on after 1.0 (we don't plan to add many more features before 1.0 — mostly fix the bugs, make the current Blitz even more stable). But we can guide you if you'd like to work on that!

**Reply by zwl1619** [reply]
@beerose Will update Next.js to 12 before 1.0?

**Reply by flybayer** [reply]
@zwl1619 yes
